src/components/repo-list/RepoList.js

```diff
@@ -44,12 +47,32 @@ const Repo = ({ repo, user }) => {
 };
 
 export const RepoList = ({ repos, user }) => {
+  const [repoFilter, setRepoFilter] = useState('lastUpdated');
+
+  const onFilterChange = (event) => {
+    const filterValue = event.target.value;
+    setRepoFilter(filterValue);
+  };
+
+  const SelectFilter = () => (
+    <Flex justifyContent="flex-end" mb="12px">
+      <Select fontSize="15px" onChange={onFilterChange} w="160px">
+        <option value="lastUpdated">Last Updated</option>
+        <option value="alphabetical">Alphabetical</option>
+        <option value="numStars">Star Count</option>
+      </Select>
+    </Flex>
+  );
+
   return repos.length ? (
     <Box m="40px 0" textAlign="left">
-      <Heading fontSize="28px" mb="24px" textAlign="center">
+      <Heading fontSize="28px" mb="16px" textAlign="center">
```

Comment:
minor question.

```
{user}'s Repositories
```

should the R in Repositories be capitalized? 

Also in the repo page, similar looking text is a link to the actual github user page. Can we add that functionality in this PR?

Reply:
I think as a title it makes sense to be capitalized. If we want to make it consistent with commits list, we can always change in different PR

For second thing, I think that should go in separate PR since this one has taken a long time